src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -1346,8 +1348,7 @@ public boolean isPrimaryThread()
 	@Override
 	public @NotNull Component motd()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return LegacyComponentSerializer.legacySection().deserialize(MOTD);
```

Comment:
I think this should be the other way around.
Aka storing `MOTD` as a `Component` internally and serializing the string version.